quartz/plugins/transformers/links.ts

```diff
@@ -73,6 +76,17 @@ export const CrawlLinks: QuartzTransformerPlugin<Partial<Options> | undefined> =
                   ) as SimpleSlug
                   outgoing.add(simple)
                   node.properties["data-slug"] = simple
+
+                  const isMissing = !transformOptions.allSlugs.includes(
```

Comment:
this won't work directly because `allSlugs` doesn't account for folder pages (i see that you made a custom case for tags) that get automatically generated. would be happy to hear a proposal about how to surface plugin created paths to `ctx.allSlugs`

Reply:
Ok! I didn't know about folder pages (& plugin created paths), I will draft this PR, read more documentation, and see when I have time to propose something :)